src/target/intrin_rule.cc

```diff
@@ -83,6 +83,8 @@ TVM_REGISTER_GLOBAL("tvm.intrin.rule.default.ceil").set_body(DispatchPureExtern<
 
 TVM_REGISTER_GLOBAL("tvm.intrin.rule.default.round").set_body(DispatchPureExtern<FloatSuffix>);
 
+TVM_REGISTER_GLOBAL("tvm.intrin.rule.default.clz").set_body(DispatchPureExtern<Direct>);
```

Comment:
clz is not a default function, so i am not sure if direct will work. Perhaps we should not do default dispatch rule but instead define for each specific backend

Reply:
ok I thought this default is required, but it doesn't seem so indeed. Removed.